editor/import/resource_importer_scene.cpp

```diff
@@ -48,6 +48,8 @@
 #include "scene/resources/surface_tool.h"
 #include "scene/resources/world_margin_shape_3d.h"
 
+#include <cwctype>
```

Comment:
What is this used for? Other than trying to not use c++17 stl, the pr looks acceptable.

Reply:
I used if for [iswalpha](https://en.cppreference.com/w/cpp/string/wide/iswalpha). But you're right, it doesn't really fit here. I replaced it with `is_alpha` (that does a simple check), which I added to `ustring.h` next to `is_symbol`. Is it looks ok now?